tests/cobs/src/main.c

```diff
@@ -0,0 +1,191 @@
+/* Copyright 2011, Jacques Fortier. All rights reserved.
```

Comment:
Is this test written by You or just reused ?

Reply:
As described in the relevant commit,. the file was renamed and converted to ztest.